packages/jest-expo-enzyme/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "expo-module-scripts/tsconfig.base",
+  "compilerOptions": {
+    "target": "es2015",
+    "lib": ["es2015"],
+    "strict": true,
+    "module": "commonjs",
+    "declarationMap": true,
```

Comment:
seems like we could remove this (also from the puppeteer tsconfig). do they actually help? we already generate source maps for the code that actually runs.

 we probably should have a separate base tsconfig for node modules (`tsconfig.node.json`) somewhere in the repo that babel-preset-expo, eslint-config-universe, etc can use